tests/CacheManagerTest.php

```diff
@@ -37,8 +37,6 @@ public function testConstructor() {
 		\WP_Mock::expectActionAdded( 'elementor/editor/after_save', array( $cache, 'flush_fcgi_cache' ) );
 		\WP_Mock::expectActionAdded( 'fl_builder_after_save_layout', array( $cache, 'flush_fcgi_cache' ) );
 		\WP_Mock::expectActionAdded( 'wp_ajax_edit_theme_plugin_file', array( $cache, 'flush_cache' ), PHP_INT_MAX );
-		\WP_Mock::expectActionAdded( 'woocommerce_after_edit_attribute_fields', array( $cache, 'flush_fcgi_cache' ) );
-		\WP_Mock::expectActionAdded( 'init', array( $cache, 'nav_init' ) );
```

Comment:
This one should stay. It's for the navcache.

Reply:
Ups, my bad with rebase.